src/database/models/user.js

```diff
@@ -1,14 +1,48 @@
-module.exports = (sequelize, DataTypes) => {
-  const User = sequelize.define('User', {
-    first_name: {
-      type:  DataTypes.STRING,
+import bcrypt from 'bcrypt';
+
+const salt = bcrypt.genSaltSync(parseInt(process.env.SALT_ROUNDS));
+parseInt
```

Comment:
Kindly remove this

Reply:
@ladaposamuel

Reply:
it wont work without it, the error massage was that salt rounds should be a number

Reply:
Line 4 I mean I removed it and it works